albumaction/albumact.rb

```diff
@@ -0,0 +1,83 @@
+require 'json'
+require_relative '../music.rb'
+
+class Albumaction
+    def get_music_album
+        file_path = './album_data/music.json'
+        if File.exist?(file_path)
+            file_content = File.read(file_path)
+            music_data = JSON.parse(file_content)
+            if File.zero?(file_path)
+                puts "Empty album"
+            else
+                file_content = File.read(file_path)
+                music_data = JSON.parse(file_content)
+                music_data.each do |album|
+                    puts "Published date: #{album['publish_date']} ---- archived: #{album['archived']} ----- on spotify : #{album['on_spotify']}"
+                  end
+            end
+        else
+            puts "No album available"
+        end
+    end
```

Comment:
@ZewdieMc Good job and nice implementation in Albumaction class 🎉

Reply:
Hello @byiringiroscar can you please fix the linter and Rspec tests error?.....the conflicts has been already resolved so after fixing the error you can merge it. 😃